plugins/linkattacher/include/GazeboYarpPlugins/linkattacherserverimpl.h

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright (C) 2013-2015 Fondazione Istituto Italiano di Tecnologia RBCS & iCub Facility & ADVR
+ * Authors: see AUTHORS file.
+ * CopyPolicy: Released under the terms of the LGPLv2.1 or any later version, see LGPL.TXT or LGPL3.TXT
+ */
+
+#ifndef YARPGAZEBO_LINKATTACHERSERVERIMPL
+#define YARPGAZEBO_LINKATTACHERSERVERIMPL
+
+#include "gazebo/gazebo.hh"
+#include <gazebo/physics/Model.hh>
+#include <gazebo/physics/World.hh>
+#include <gazebo/physics/Joint.hh>
+#include <gazebo/physics/Link.hh>
+#include <gazebo/physics/PhysicsEngine.hh>
+#include <gazebo/common/Events.hh>
+
+#include <yarp/os/Network.h>
+#include <yarp/os/RpcServer.h>
+#include <yarp/os/Property.h>
+#include <yarp/os/Log.h>
+#include <yarp/os/LogStream.h>
+#include <GazeboYarpPlugins/common.h>
+#include <GazeboYarpPlugins/ConfHelpers.hh>
+
+#include <LinkAttacherServer.h>
+
+const std::string LogPrefix = "LinkAttacher: ";
```

Comment:
A trick. I recently realized that yarp logging streams already add a space if you concatenate things with `<<`. This means that you can remove this trailing space.

Reply:
Done